### ISSUE TITLE
Seedvault - Perhaps some useful code

### ISSUE THREAD

**Issue by Donkey-Doug** [NeoApplications/neo-backup#507]
This is the backup solution that is baked into lineageos:

https://github.com/seedvault-app/seedvault

Perhaps you could learn from their methods/code.

**Comment by BlueDrink9** [comment]
I'm sure your intentions are good, but if you read the FAQ you'll see they are well aware of seedvault! This issue doesn't contribute anything, so perhaps would be best if you close it

**Comment by BlueDrink9** [comment]
(Alternatively, give some specific examples of code from seedvault repo you think would be specifically useful to Neobackup - assuming its licence permits reuse)

**Comment by machiav3lli** [comment]
We have clear distinction of the two projects in the FAQ.
tldr; we have fully functions and go different ways handling backups. Seedvault uses System+ADB and respects the APIs while we don't.
